packages/contracts-bridge/contracts/test/NFTAccountant.t.sol

```diff
@@ -85,41 +143,120 @@ contract NFTAccountantTest is Test {
         assertEq(accountant.totalMinted(asset), _prevTotal + amnt);
     }
 
-    function test_recordCheck() public {
+    function recordCheck(
```

Comment:
this is all duplicated code with function above. de-duplicate by having the no-argument version invoke this version?

Reply:
both of these functions read local state in an unpleasant way that makes them dependent on the context in which they're invoked. Try to remove deps on test state by converting to argument?

E.g. these tests are very unclear, and brittle to changes in `recordCheck` and in local state

![Screenshot 2022-11-21 at 9 36 14 AM](https://user-images.githubusercontent.com/10149425/203081665-2510efab-7ad5-443b-9e32-14ad10817a46.png)